front/app/containers/InitiativesShow/VoteControl/ProposedNotVoted.tsx

```diff
@@ -156,124 +156,123 @@ const disabledMessages: {
   notPermitted: messages.votingNotPermitted,
 };
 
-class ProposedNotVoted extends PureComponent<Props & { theme: any }> {
-  handleOnVote = () => {
-    this.props.onVote();
+const ProposedNotVoted = ({
+  onVote,
+  initiative,
+  initiativeSettings: { voting_threshold, threshold_reached_message },
+  disabledReason,
+}: Props) => {
+  const theme: any = useTheme();
+  const handleOnVote = () => {
```

Comment:
Why not pass directly `onVote`? No need to create an intermediate method. :)

Reply:
💯 thought about it too but forgot. Changed it.